### PR TITLE
fragment events

### PR DESCRIPTION
**Pull request by kyeotic**
This is a minimal working set for capturing/bubbling events on Fragements, as discussed in #12051. @bsidelinger912 helped me with this (thank you Ben!)

I'm opening this PR now to start a discussion, and to see if I am on the right track. There are still failing tests as a result of this change but I want to make sure this is something the team is still interested in before I invest more time to fix them.

I have proved this out with the included unit test, as well as with manual confirmation with a built version in the test fixture (which I didn't commit).

### CODE REVIEW

File: packages/react-reconciler/src/ReactChildFiber.js [hunk 1]
```diff
@@ -516,6 +516,17 @@ function ChildReconciler(shouldTrackSideEffects) {
           created.return = returnFiber;
           return created;
         }
+        case REACT_FRAGMENT_TYPE: {
+          console.log('fargment child creating');
```
Comment (jacobp100) [inline packages/react-reconciler/src/ReactChildFiber.js]:
```suggestion
```

Comment (kyeotic) [reply packages/react-reconciler/src/ReactChildFiber.js]: removed

**Comment by trueadm** [comment]
This isn't something we're looking to support with the existing event system, so I'm going to close this PR for now.

**Comment by kyeotic** [comment]
@trueadm That's disappointing. Are there plans to support it in a future event system, or should I read that as React never intends to support this kind of abstraction?

**Comment by trueadm** [comment]
@kyeotic It's something I'm actively investigating as part of our experimention with the new event system, although there's nothing planned in the short-term.

**Pull request closed without merging**